client/js/transpecos.js

```diff
@@ -54,32 +54,35 @@
 
 	var _reactDom2 = _interopRequireDefault(_reactDom);
 
-	function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+	var _navbar = __webpack_require__(178);
```

Comment:
`git rm client/js/transpecos.js` and add `clilent/js/transpecos.js` to `.gitignore` -- you should not be committing this to source control; rather it should be rebuilt via webpack.